core/application/app_test.go

```diff
@@ -2,15 +2,18 @@ package application_test
 
 import (
 	"context"
+	"encoding/base64"
 	"errors"
 	"testing"
 	"time"
 
 	"github.com/golang/mock/gomock"
 	. "github.com/onsi/gomega"
 	"github.com/spf13/afero"
+	"gopkg.in/yaml.v2"
```

Comment:
I'd probably use https://pkg.go.dev/sigs.k8s.io/yaml instead of v2 of go-yaml

Reply:
Changed to use this wrapper.